File: BE_QuanLyTro/src/main/java/ut/edu/be_quanlytro/Dto/Request/ContractTemplateRequest.java
package ut.edu.be_quanlytro.Dto.Request;

import lombok.Getter;
import lombok.Setter;

@Getter @Setter
public class ContractTemplateRequest {
    private String name;

    // 4 trường nội dung mới thay thế cho content cũ
    private String rentalContent;   // Nội dung thuê phòng
    private String landlordDuty;    // Trách nhiệm bên A
    private String tenantDuty;      // Trách nhiệm bên B
    private String executionTerms;  // Điều khoản thực hiện
}

File: Flutter_QuanLyTro/lib/data/models/request/contract_template_request.dart
class ContractTemplateRequest {
  final String name;
  final String rentalContent;
  final String landlordDuty;
  final String tenantDuty;
  final String executionTerms;

  ContractTemplateRequest({
    required this.name,
    required this.rentalContent,
    required this.landlordDuty,
    required this.tenantDuty,
    required this.executionTerms,
  });

  Map<String, dynamic> toJson() {
    return {
      "name": name,
      "rentalContent": rentalContent,
      "landlordDuty": landlordDuty,
      "tenantDuty": tenantDuty,
      "executionTerms": executionTerms,
    };
  }
}